server/src/resolvers/Task.js

```diff
@@ -0,0 +1,13 @@
+function __resolveType(user, context, info) {
+  const newbieUniqueProps = ['notes'];
+
+  if (newbieUniqueProps.some(prop => user.hasOwnProperty(prop))) {
+    return "NewbieTask";
```

Comment:
I've noticed inconsistency when using quotes in this file. I think we use single quotes everywhere 🤔? 

https://github.com/BuildIt-Poland/buddy-app/blob/31568ec21c7389073b924720509e18bd351b2249/.prettierrc#L4

Wonder if this file is linted as it should be picked-up by prettier and fixed automatically when committing using git hook.